.github/workflows/changelog-existence.yml

```diff
@@ -19,7 +19,6 @@ name: Check Changelog Entry
 on:
   pull_request_target:
     types: [opened, reopened, labeled, unlabeled, synchronize]
-    paths-ignore: ['.changes/**', '.github/**', 'tests/**', 'third-party-stubs/**', '**.md', '**.yml']
```

Comment:
Apparently if all changes are in skipped paths and the test is required, it never runs, hence never passes.